uisdk/proguard-rules.pro

```diff
@@ -0,0 +1,32 @@
+# Add project specific ProGuard rules here.
+# You can control the set of applied configuration files using the
+# proguardFiles setting in build.gradle.
+#
+# For more details, see
+#   http://developer.android.com/guide/developing/tools/proguard.html
+
+# If your project uses WebView with JS, uncomment the following
+# and specify the fully qualified class name to the JavaScript interface
+# class:
+#-keepclassmembers class fqcn.of.javascript.interface.for.webview {
+#   public *;
+#}
+
+# Uncomment this to preserve the line number information for
+# debugging stack traces.
+#-keepattributes SourceFile,LineNumberTable
+
+# If you keep the line number information, uncomment this to
+# hide the original source file name.
+#-renamesourcefileattribute SourceFile
+-keep class **.R$raw**
+-keep class **.R$raw$* {
+    <fields>;
+}
+-keepclassmembers class **.R$raw {
+    public static int *;
+}
+-keep class  tech.dojo.pay.uisdk.data.** { *; }
+-keep class  tech.dojo.pay.uisdk.entities.** { *; }
+-keepclassmembers class tech.dojo.pay.uisdk.data.** { *; }
```

Comment:
Do you need `{ *; }`?

Reply:
as we talked about that one we need it as i want to keep every class and every method inside that class